ext_mongodb.php

```diff
@@ -929,21 +929,21 @@ public function __construct(int $readPreference, array $tagSets = null, array $o
 			$this->_setReadPreferenceTags( $newTagSets );
 		}
 
-		if ( array_key_exists( 'maxStalenessMS', $options ) )
+		if ( array_key_exists( 'maxStalenessSeconds', $options ) )
 		{
-			$maxStalenessMS = (int) $options['maxStalenessMS'];
+			$maxStalenessSeconds = (int) $options['maxStalenessSeconds'];
 
-			if ( $maxStalenessMS < 0 )
+			if ( $maxStalenessSeconds < 0 )
 			{
-				Utils::throwHippoException( Utils::ERROR_INVALID_ARGUMENT, "Expected maxStalenessMS to be >= 0, {$maxStalenessMS} given" );
+				Utils::throwHippoException( Utils::ERROR_INVALID_ARGUMENT, "Expected maxStalenessSeconds to be >= 0, {$maxStalenessSeconds} given" );
```

Comment:
The new spec says it should be 90 or more - I suppose we want to make that check here in the code as well?